embc-app/DataInterfaces/ViewModelConversions.EvacueeRegistration.cs

```diff
@@ -73,7 +73,7 @@ public static ViewModels.RegistrationSummary ToSummaryViewModel(this Models.Db.E
                 IncidentTask = fullViewModel.IncidentTask,
                 HostCommunity = fullViewModel.HostCommunity,
                 Active = fullViewModel.Active ?? false,
-                HasFollowUpDetails = !string.IsNullOrWhiteSpace(fullViewModel.FollowUpDetails),
+                HasFollowUpDetails = !string.IsNullOrWhiteSpace(fullViewModel.InternalCaseNotes),
```

Comment:
@severinbeauvais do you want this to be changed to HasInternalCaseNotes?

Reply:
Oh yes, I missed that ticket.